tests/storage/storage.go

```diff
@@ -1056,10 +1044,12 @@ var _ = SIGDescribe("Storage", func() {
 			// Not a candidate for NFS because local volumes are used in test
 			It("[test_id:1015]should be successfully started", func() {
 				// Start the VirtualMachineInstance with the PVC attached
-				vmi = tests.NewRandomVMIWithPVC(dataVolume.Name)
 				// Without userdata the hostname isn't set correctly and the login expecter fails...
-				tests.AddUserData(vmi, cloudInitName, "#!/bin/bash\necho 'hello'\n")
-
+				vmi = libvmi.New(
+					libvmi.WithResourceMemory("256Mi"),
+					libvmi.WithPersistentVolumeClaim("disk0", dataVolume.Name, v1.DiskBusVirtio),
+					libvmi.WithCloudInitNoCloudUserData("#!/bin/bash\necho 'hello'\n", true),
```

Comment:
can you make this string a const and name it cirrosHack? 

And we need to find a way to get rid of this in near future, maybe switch to Alpine? But the switch of OS is an idea for another PR.